lib/DataSheet.js

```diff
@@ -603,6 +669,8 @@ DataSheet.propTypes = {
   onChange: _propTypes2.default.func,
   onCellsChanged: _propTypes2.default.func,
   onContextMenu: _propTypes2.default.func,
+  onSelect: _propTypes2.default.func,
+  selected: _propTypes2.default.object,
```

Comment:
You can define a shape here for better type checking : 
https://reactjs.org/docs/typechecking-with-proptypes.html
```
optionalObjectWithShape: PropTypes.shape({
    color: PropTypes.string,
    fontSize: PropTypes.number
  }),
```